dashboard/app/models/user.rb

```diff
@@ -109,6 +109,8 @@ class User < ActiveRecord::Base
     races
     using_text_mode
     last_seen_school_info_interstitial
+    ui_tip_dismissed_homepage_header
+    ui_tip_dismissed_teacher_courses
```

Comment:
I want to keep them up through Hour of Code (remove January) or when 90% of teachers have seen it. So we can take them down after whichever comes first.